vhd/packer/configure-windows-vhd.ps1

```diff
@@ -202,7 +202,8 @@ function Install-WindowsPatches {
             # then you can get download links by searching for specific KBs at http://www.catalog.update.microsoft.com/home.aspx
 
             # Find a specific patch at https://www.catalog.update.microsoft.com/Search.aspx?q=kb5005625
-            $patchUrls = @()
+            # KBKB5015880 Contains 2022 7C patches for Windows server 2019

```

Comment:
```suggestion
            # KB5015880 Contains 2022 7C patches for Windows server 2019
```

Reply:
really not necessary and since ci is mostly complete, could update this when we update the push the vhd updates.